### PR TITLE
Add back removed files - 2nd try

### PR DESCRIPTION
**Pull request by aparajit-pratap**
### Purpose

Add back removed files - this time with empty XML root element.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

**Review by QilongTang — APPROVED** [review]

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -168,8 +168,8 @@
     <Copy SourceFiles="$(ProjectDir)\BuiltInAndOperators\BuiltIn.xml" DestinationFolder="$(OutputPath)\$(UICulture)" />
     <Copy SourceFiles="$(ProjectDir)\BuiltInAndOperators\BuiltIn.Migrations.xml" DestinationFolder="$(OutputPath)" />
     <!--Enable these if/when we need to keep old binaries for patch installer compatability-->
-    <!--<Copy SourceFiles="@(LegacyBinariesToBinFolder)" DestinationFolder="$(OutputPath)" />
-    <Copy SourceFiles="@(LegacyBinariesToNodesFolder)" DestinationFolder="$(OutputPath)\nodes\" />-->
+    <Copy SourceFiles="@(LegacyBinariesToBinFolder)" DestinationFolder="$(OutputPath)" />
+    <Copy SourceFiles="@(LegacyBinariesToNodesFolder)" DestinationFolder="$(OutputPath)\nodes\" />
```
Comment (QilongTang) [inline src/DynamoCore/DynamoCore.csproj:172]: I thought this change already included in your last PR.. Maybe some git diff bug

**Review by mjkkirschner — APPROVED** [review]

**Comment by mjkkirschner** [comment]
@aparajit-pratap can you verify a few of those tests that failed pass now locally?

**Comment by aparajit-pratap** [comment]
> @aparajit-pratap can you verify a few of those tests that failed pass now locally?

Yes, I did verify that.

**Comment by aparajit-pratap** [comment]
DynamoSelfServe failing due to 2 flaky tests, which is unrelated to this PR - merging

**Pull request merged**